index.php

```diff
@@ -122,11 +98,12 @@
                 <li>All Rights Reserved</li>
                 <li><a href="https://github.com/WChihWen" target="_blank">Chih Wen Wang</a></li>
                 <li><a href="https://github.com/jmgarza94" target="_blank">Joey Garza</a></li>
+                <li><a href="https://github.com" target="_blank">John Meyer</a></li>
```

Comment:
This <li> link does not need to be in the file any longer, as it does not go to a specific repository.